modules/userService.js

```diff
@@ -127,3 +127,35 @@ module.exports.setToken = (token, inputEmail) => {
 		}
 	});
 };
+
+module.exports.validateToken = (token, inputEmail) => {
+	return new Promise(function(resolve, reject) {
+		try {
```

Comment:
since mongoose uses a callback that has an err object passed to it, the try...catch block doesn't seem necessary. instead just do if(err){`reject()`}else{`resolve()`} inside the findOne callback